trade-finance-manager-api/src/v1/rest-mappings/facility.js

```diff
@@ -1,18 +1,18 @@
-const mapFacility = require('./mappings/facilities/mapFacility');
+const mapFacilitySnapshot = require('./mappings/facilities/mapFacilitySnapshot');
 const mapFacilityTfm = require('./mappings/facilities/mapFacilityTfm');
 const mapGefFacility = require('./mappings/gef-facilities/mapGefFacility');
 const isGefFacility = require('./helpers/isGefFacility');
 
 const facilityMapper = (facility, dealSnapshot, dealTfm) => {
```

Comment:
Would be nice to add some documentation whilst we're here and in `mapFacilities`, not a blocker

Reply:
Good idea - I'll merge this PR as is, and I hope to do another small PR next with a few more improvements and I'll include documentation for those files then. 👍